Sources/Crashes/CrashCollection.swift

```diff
@@ -19,61 +19,86 @@
 import Foundation
 import MetricKit
 
+public enum CrashCollectionPlatform {
+    case iOS, macOS, macOSAppStore
```

Comment:
`CrashCollection` itself (MetricKit adapter) only uses `iOS` and `macOSAppStore`, but this enum is consumed by `CrashReportSender` that is the common interface to our backend and handles all 3 cases.